### PR TITLE
checking for generated docs

### PR DESCRIPTION
**Pull request by chrislovecnm**
This will fail until I rebase with a doc update.

Fixes https://github.com/kubernetes/kops/issues/1688

Depends on https://github.com/kubernetes/kops/pull/2029

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2028)

### CODE REVIEW

**Comment by justinsb** [comment]
LGTM - please self merge when ready

**Comment by chrislovecnm** [comment]
Travis does not like mkdir command... will fix

File: hack/verify-gendocs.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. $(dirname "${BASH_SOURCE}")/common.sh
+
+TMP_DOCS="${KUBE_ROOT}/.build/docs"
+rm -rf $TMP_DOCS
+mkdir $TMP_DOCS
```
Comment (chrislovecnm) [inline hack/verify-gendocs.sh]: This command is not working, need to fix

**Comment by justinsb** [comment]
Just needs you to apply the above fix I believe @chrislovecnm

**Comment by chrislovecnm** [comment]
Yeps ... this is on my todo list :)

**Comment by chrislovecnm** [comment]
Closing because we did https://github.com/kubernetes/kops/issues/2403

**Pull request closed without merging**